libbeat/dashboards/import_dashboards.go

```diff
@@ -211,7 +211,7 @@ func (imp Importer) CreateIndex() error {
 			},
 		})
 	if err != nil {
-		fmt.Printf("fail to set the mapping. Error: %s\n", err)
+		fmt.Printf("failed to set the mapping. Error: %s\n", err)
```

Comment:
Three are a few other places where it prints directly to `stdout` for error besides this. The script overall is not terribly consistent about this sort of thing.